seL4-platforms/platforms.yml

```diff
@@ -58,6 +58,14 @@ platforms:
     req: [hifive] # , hifive1]
     march: rv64imac
 
+  STAR64:
+    arch: riscv
+    modes: [64]
+    smp: [64]
+    platform: star64
+    req: [star64]
+    march: rv64imac
```

Comment:
Not sure what `march` does exactly here, `march: rv64gcb` would be more accurate for the Star64. The only use of `march` I can see is here https://github.com/seL4/ci-actions/blob/607634d9c99af003055024f8bd11c47b4bdc757e/seL4-platforms/platforms.py#L183-L192, where it seems to be ignoring `march` anyways.

Reply:
I might be wrong, but I think the only remaining use of `march` is for grouping in CI matrices. We could try to eliminate it and see what happens..